GPL/nativeBuildProperties.gradle

```diff
@@ -43,10 +43,10 @@ if (org.gradle.internal.os.OperatingSystem.current().isWindows()) {
 		project.ext.WINVER = "0x0601"
 	
 		// Rely on vcvars script to supply SDK versions
-		def COMMAND = "cmd /c ${VISUAL_STUDIO_VCVARS_CMD} > nul && cmd /c echo"
-		project.ext.MSVC_SDK_VERSION = "${COMMAND} %WINDOWSSDKVERSION%".execute().text.trim().replace('\\', '')
+		def COMMAND = "cmd /v /c \"${VISUAL_STUDIO_VCVARS_CMD} > nul && cmd /c echo"
```

Comment:
What does that escaped double-quote do?

Reply:
Without that, I cannot execute command in regular CMD. Generated command was executed in Gradle, but not in my cmd shell. It outputs just `%WINDOWSSDKVERSION%`, but work correctly if I split code on `&&`. I make that change mostly for better debuggability of the code, since I can drop command in the CLI and validate that it works. These quotation was initially my hope to make it run in AzureDevOps, but it start working only after `/v`. I did not check is AzureDevOps will work with `/v` but without quotes.

I test that on `[Version 10.0.18362.295], [Version 10.0.17763.678] and [Version 10.0.17134.950]`

Reply:
/cc @ryanmkurtz

Reply:
@ryanmkurtz  They are matched, I add quote to the end of "${COMMAND} !WINDOWSSDKVERSION!\"" you just miss that. If whole command would be constructed like that
```
"cmd /v /c \"${VISUAL_STUDIO_VCVARS_CMD} > nul && cmd /c echo !WINDOWSSDKVERSION!\""
```
it will be easier to understand, but closing quote almost invisible due to appearing in the end of string.

Reply:
Just `/v` helps me. I push updated changes.

Reply:
Yes, should I update code?